q2_stats/_transformer.py

```diff
@@ -62,14 +62,18 @@ def _4(obj: pd.DataFrame) -> TabularDataResourceDirFmt:
 
         metadata['name'] = col
         metadata['type'] = schema_dtype
-        metadata_resource.schema.add_field(source=metadata)
 
-    metadata_resource.data = 'data.ndjson'
-    metadata_resource.format = 'ndjson'
+        metadata_obj.append(metadata)
+
+    metadata_dict = dict({'schema': metadata_obj})
+    metadata_dict['format'] = 'ndjson'
+    metadata_dict['path'] = 'data.ndjson'
+    json.dumps(metadata_dict)
 
     dir_fmt = TabularDataResourceDirFmt()
 
     dir_fmt.data.write_data(obj, pd.DataFrame)
-    metadata_resource.to_json(str(dir_fmt.path/'dataresource.json'))
+    with open(dir_fmt.path/'dataresource.json', 'w') as fh:
```

Comment:
Probably spaces around the / right? Since it's an operator? Idk though since lint isn't complaining.

Reply:
Just tested this and it seems to work both ways 🤷‍♀️ @ebolyen do you have a preference?

Reply:
I would agree on spaces, but if flake8 doesn't care, then I don't care either :-)